port/common/omrportcontrol.c

```diff
@@ -72,8 +72,36 @@ omrport_control(struct OMRPortLibrary *portLibrary, const char *key, uintptr_t v
 			return (int32_t)PPG_mem_mem32_subAllocHeapMem32.suballocator_commitSize;
 		}
 		return 0;
+	} else if (0 == strcmp(OMRPORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE, key)) {
+		if (0 != value) {
+			/* IncrementSize is immutable. It can only be set once. */
+			if (0 == PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize) {
+				/* Round up the increment size to the page size and set it to global variable. */
+				uintptr_t pageSize = portLibrary->vmem_supported_page_sizes(portLibrary)[0];
+				uintptr_t roundedIncrementSize = pageSize * (value / pageSize);
+				if (roundedIncrementSize < value) {
+					roundedIncrementSize += pageSize;
+				}
+				PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize = roundedIncrementSize;
+			} else {
+				return 1;
+			}
+		} else {
+			return (int32_t)PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize;
+		}
+		return 0;
+	} else if (0 == strcmp(OMRPORT_CTLDATA_ALLOCATE32_QUICK_ALLOC, key)) {
+		if (FALSE == value) {
```

Comment:
`value` is defined as `uintptr_t`, so despite key stores boolean value it should be compared with `0`: `if (0 == value) {`